src/__tests__/home.test.js

```diff
@@ -0,0 +1,14 @@
+import { Provider } from 'react-redux';
```

Comment:
- [ ] It's good you have tests in this project. However, one of your tests is showing some errors in the console. Please avoid all mistakes to make your project as approvable:

<img width="804" alt="Screen Shot 2022-10-13 at 06 53 41" src="https://user-images.githubusercontent.com/69011963/195482386-b643359a-da53-4e3f-8554-4cb67fe0eb45.png">